website/docs/community/contributing/contributing-online-community.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Online community building"
+id: "contributing-online-community"
+---
+
+### Maintaining a channel in the dbt Community Slack
+
+#### Overview
+
+The dbt Slack is the place for real time conversations with the dbt Community. Slack channels exist to for specific locations, tools, industries and methodologies. In order to make sure that every channel has dedicated attention from a committed community member, we have Community Maintainers who oversee the discussion in particular channels.
+
+#### Contribution opportunities
+
+We’re working on compiling a list of existing channels that could benefit from additional community maintainers. In the meantime, feel free to be on the lookout for opportunities to help maintain new channels that are getting spun up by keeping an eye on #channel-requests in the dbt slack.
+
+#### Sample contributions:
+
+- Karen Hsieh's [contributions](https://getdbt.slack.com/archives/C02TU2DSKND/p1661483529756289) to the #local-taipei channel are a fantastic example to learn from.
+
+#### Get started
+
+- Read the guide to [Maintaining a Slack Channel](https://docs.getdbt.com/community/maintaining-a-channel)
```

Comment:
@gwenwindflower can we double check the redirect on this for the new url under resources?